presto-main/src/test/java/com/facebook/presto/cost/TestFilterStatsCalculator.java

```diff
@@ -37,6 +37,14 @@
 {
```

Comment:
can you please add a test where stats become better?

Reply:
This test:
```
        assertExpression("NOT(x < 0e0)")
                .outputRowsCount(625) // FIXME - nulls shouldn't be restored
                .symbolStats(new Symbol("x"), symbolAssert ->
                        symbolAssert.averageRowSize(4.0)
                                .lowValue(0.0)
                                .highValue(10.0)
                                .distinctValuesCount(20.0)
                                .nullsFraction(0.4)) // FIXME - nulls shouldn't be restored
                .symbolStats(new Symbol("y"), symbolAssert -> symbolAssert.isEqualTo(yStats));
```
validates that y stats are preserved, which wasn't the case before